translations/servers.json

```diff
@@ -0,0 +1,1553 @@
+   "hsb": "Finska",
+   "oc": "Finlàndia",
+   "pt_BR": "Finlândia",
+   "sk": "Fínsko",
+   "sq": "Finlanda",
+   "uk": "Фінляндія",
+   "ru": "Финляндия",
+   "ja": "フィンランド",
+   "fr": "Finlande",
+   "de": "Finnland",
+   "el": "Φινλανδία",
+   "it": "Finlandia",
+   "co": "Finlandia",
+   "cy": "Y Ffindir",
+   "dsb": "Finska"
+  },
+  "cities": [
+   {
+    "city": "Helsinki",
+    "languages": {
+     "zh_TW": "赫爾辛基",
+     "pt_BR": "Helsinque",
+     "ru": "Хельсинки",
+     "uk": "Гельсінкі",
+     "el": "Ελσίνκι",
+     "ja": "ヘルシンキ"
+    }
+   }
+  ]
+ },
+ {
+  "countryCode": "fr",
+  "languages": {
+   "zh_TW": "法國",
+   "ru": "Франция",
+   "nl": "Frankrijk",
+   "it": "Francia",
+   "oc": "França",
+   "pt_BR": "França",
+   "ja": "フランス",
+   "de": "Frankreich",
+   "el": "Γαλλία",
+   "uk": "Франція",
+   "sq": "Francë",
+   "sk": "Francúzsko",
+   "co": "Francia",
+   "cy": "Ffrainc",
+   "dsb": "Francojska",
+   "hsb": "Francoska"
+  },
+  "cities": [
+   {
+    "city": "Paris",
+    "languages": {
+     "co": "Parighji",
+     "zh_TW": "巴黎",
+     "ja": "パリ",
+     "nl": "Parijs",
+     "oc": "París",
+     "it": "Parigi",
+     "ru": "Париж",
+     "uk": "Париж",
+     "el": "Παρίσι",
+     "sk": "Paríž"
+    }
+   }
+  ]
+ },
+ {
+  "countryCode": "de",
+  "languages": {
+   "zh_TW": "德國",
+   "co": "Germania",
+   "cy": "Yr Almaen",
+   "de": "Deutschland",
+   "dsb": "Nimska",
+   "el": "Γερμανία",
+   "fr": "Allemagne",
+   "hsb": "Němska",
+   "it": "Germania",
+   "ja": "ドイツ",
+   "nl": "Duitsland",
+   "oc": "Alemanha",
+   "pt_BR": "Alemanha",
+   "ru": "Германия",
+   "sk": "Nemecko",
+   "sq": "Gjermania",
+   "uk": "Німеччина"
+  },
+  "cities": [
+   {
+    "city": "Dusseldorf",
+    "languages": {
+     "oc": "Düsseldorf",
+     "pt_BR": "Düsseldorf",
+     "ru": "Дюссельдорф",
+     "sk": "Düsseldorf",
+     "sq": "Dyzeldorfi",
+     "uk": "Дюссельдорф",
+     "el": "Ντύσσελντορφ",
+     "zh_TW": "杜塞爾多夫",
+     "co": "Düsseldorf",
+     "cy": "Düsseldorf",
+     "de": "Düsseldorf",
+     "dsb": "Düsseldorf",
+     "en": "Düsseldorf",
+     "en_CA": "Düsseldorf",
+     "en_GB": "Düsseldorf",
+     "fr": "Düsseldorf",
+     "hsb": "Düsseldorf",
+     "it": "Düsseldorf",
+     "ja": "デュッセルドルフ",
+     "nl": "Düsseldorf"
+    }
+   },
+   {
+    "city": "Frankfurt",
+    "languages": {
+     "oc": "Francfòrt de Men",
+     "ru": "Франкфурт-на-Майне",
+     "sk": "Frankfurt nad Mohanom",
+     "sq": "Frankfurti mbi Main",
+     "uk": "Франкфурт-на-Майні",
+     "zh_TW": "美茵河畔法蘭克福",
+     "nl": "Frankfurt am Main",
+     "de": "Frankfurt am Main",
+     "en": "Frankfurt am Main",
+     "co": "Francuforte nant'à u Menu",
+     "cy": "Frankfurt am Main",
+     "dsb": "Frankfurt nad Mainom",
+     "el": "Φραγκφούρτη",
+     "fr": "Francfort-sur-le-Main",
+     "hsb": "Frankobrod nad Mohanom",
+     "it": "Francoforte sul Meno",
+     "ja": "フランクフルト・アム・マイン"
+    }
+   }
+  ]
+ },
+ {
+  "countryCode": "hk",
```

Comment:
This one, for example, doesn't have a `en` name. How do you fall back to English?

Reply:
The fallback is what we receive from mullvad.